src/test/java/spoon/support/compiler/SpoonPomTest.java

```diff
@@ -51,4 +52,20 @@ public void checkProfilesModules(String path, String[] expected, Pattern profile
 			assertEquals(expected[i], modules.get(i).getName());
 		}
 	}
+	
+	public void getSourceDirectory() throws IOException, XmlPullParserException {
+		checkSourceDirectory(
+			"src/test/resources/maven-launcher/hierarchy",
+			Paths.get("src/test/resources/maven-launcher/hierarchy/child/src").toAbsolutePath().toString()
+		);
+	}
+
+	public void checkSourceDirectory(String path, String expected) throws IOException, XmlPullParserException {
+		SpoonPom pomModel = new SpoonPom(path, null, MavenLauncher.SOURCE_TYPE.APP_SOURCE, new StandardEnvironment());
+
+		SpoonPom childModel = pomModel.getModules().get(0);
+		//contract: source directory is derived from parent pom.xml if not declared in the current
+		// (childModel) SpoonPom
+		assertEquals(expected, childModel.getSourceDirectories().get(0).getAbsolutePath(), expected);
```

Comment:
expected should be the second argument, not the last
you can remove the first one, the message, not relevant here

Reply:
Fixed according to the hover definition :+1: 
![image](https://user-images.githubusercontent.com/18282288/92598349-9400f000-f2a0-11ea-9037-a62d0fa35cc3.png)